modules/monitoring-setting-audit-log-levels-for-the-prometheus-adapter.adoc

```diff
@@ -0,0 +1,117 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_content-type: PROCEDURE
+[id="setting-audit-log-levels-for-the-prometheus-adapter_{context}"]
+= Setting audit log levels for the Prometheus Adapter
+
+[role=_abstract]
+In default platform monitoring, you can configure the audit log level for the Prometheus Adapter.
```

Comment:
When I checked, the capitalization of PA online varies from one source to another. 
Do the following suggestions seem right to you?
- If PA is a proper noun, one drops the definite article, like this:  "...for Prometheus Adapter"? 
- Otherwise, for an ordinary noun, one keeps the definite article and drops the capitalization of adapter, like this: "for the Prometheus adapter"

Reply:
Thanks, good catch. It should be Prometheus Adapter, so without the article. I'll update it.